fix-point/src/Refactor.hs

```diff
@@ -1,39 +1,42 @@
 module Rename where 
 
-import qualified Data.Text as Universum
-import qualified Data.Text.IO as Universum
+import Data.Text (Text)
+import Data.Semigroup
```

Comment:
According to our style-guide all unqualified imports should have explicit import lists. So here you should export operator like this:

```haskell
import Data.Semigroup ((<>))
```